src/libNode/Node.cpp

```diff
@@ -1785,55 +1817,72 @@ bool Node::ProcessTxnPacketFromLookup([[gnu::unused]] const bytes& message,
            0) ||
           m_justDidFallback))) {
       SHA2<HashType::HASH_VARIANT_256> sha256;
-      sha256.Update(message2);  // message hash
+      sha256.Update(message);  // message hash
       bytes msg_hash = sha256.Finalize();
       lock_guard<mutex> g2(m_mutexTxnPacketBuffer);
-      m_txnPacketBuffer.emplace(msg_hash, message2);
+      m_txnPacketBuffer.emplace(msg_hash, message);
       return true;
     }
   }
 
+  // for shard:
+  // 1. DS epoch: lookup dispatch in new DS epoch, buffer and dispatch when mb
+  // finish, so normally first DS epoch won't have txns
+  // 2. FB epoch: saying the 2nd epoch after ds epoch, lookup dispatch txn upon
+  // mb, they distribute right away until mb consensus started. During the mb
+  // submitting mb, all the packet received should be buffered
```

Comment:
This and next line sound confusing. Anyway I think we can re-phrase these comments more clearly (can discuss offline). Also, these will be documented in detail in the ZIP.

Reply:
@haichuan